src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/shipping.html.twig

```diff
@@ -24,7 +24,19 @@
  *#}
 
 {% if not orderForViewing.virtual %}
-  <table class="table">
+
+    {% if orderForViewing.shipping.giftMessage %}
+      <div class="row col-lg-12">
+        <label>
+          {{ 'Message:'|trans({}, 'Admin.Global') }}
```

Comment:
```suggestion
          {{ 'Message:'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Reply:
Sure about that ? I copied the legacy order page wording.

Reply:
100% SURE 💪

<img width="609" alt="Capture d’écran 2020-03-12 à 14 48 23" src="https://user-images.githubusercontent.com/32565890/76528137-9fe63380-6470-11ea-98a0-1f07a752ef7b.png">

Reply:
Just a suggestion but wouldn't "Gift message:" be more accurate in this case?

Reply:
I'll let you guys decide and I'll update the PR after 😄

Reply:
Wordings are very basic in this page but it could be a nice improvement, indeed! And in this case, let's keep it localized in _Admin.Global_, it makes more sense as it could be reused somewhere else in the back office.

Reply:
```suggestion
          {{ 'Gift message:'|trans({}, 'Admin.Global') }}
```

Reply:
I apply your suggestion 👍